scipy/signal/fir_filter_design.py

```diff
@@ -594,3 +596,106 @@ def remez(numtaps, bands, desired, weight=None, Hz=1, type='bandpass',
     bands = np.asarray(bands).copy()
     return sigtools._remez(numtaps, bands, desired, weight, tnum, Hz,
                            maxiter, grid_density)
+
+
+def firls(numtaps, bands, desired, weight=None, fs=2):
```

Comment:
Ahh right you said `nyq=1.0` referring to Nyquist *frequency*, I was thinking of Nyquist *rate*. So changing `fs=2.`->`nyq=1.` means the other args stay the same. Will change.